apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -496,39 +765,56 @@ static void tc_fs_vfs_dup2(void)
 	ssize_t ret;
 	int fd1, fd2, len;
 
+	/* Init */
+	vfs_mount();
+
 	memset(buf, 0, sizeof(buf));
 
 	fd1 = open(filename1, O_RDWR);
-	TC_ASSERT_GEQ("open", fd1, 0);
+	TC_ASSERT_GEQ_CLEANUP("open", fd1, 0, vfs_unmount());
 
 	fd2 = open(filename2, O_RDWR | O_CREAT);
 	close(fd1);
-	TC_ASSERT_GEQ("open", fd2, 0);
+	TC_ASSERT_GEQ_CLEANUP("open", fd2, 0, vfs_unmount());
 
+	/* Testcase */
 	/* now fd1 points fd2 */
 	ret = dup2(fd2, fd1);
 	close(fd2);
-	TC_ASSERT_NEQ("dup2", ret, ERROR);
-	TC_ASSERT_GEQ("dup2", fd1, 0);
+	TC_ASSERT_NEQ_CLEANUP("dup2", ret, ERROR, vfs_unmount());
+	TC_ASSERT_GEQ_CLEANUP("dup2", fd1, 0, vfs_unmount());
 
 	len = strlen(VFS_TEST_CONTENTS_3);
 	ret = write(fd1, str, len);
 	close(fd1);
-	TC_ASSERT_EQ("write", ret, len);
+	TC_ASSERT_EQ_CLEANUP("write", ret, len, vfs_unmount());
 
 	fd2 = open(filename2, O_RDONLY);
-	TC_ASSERT_GEQ("open", fd2, 0);
+	TC_ASSERT_GEQ_CLEANUP("open", fd2, 0, vfs_unmount());
 
 	ret = read(fd2, buf, len);
+
+	TC_ASSERT_GT_CLEANUP("read", ret, 0, close(fd2));
+	TC_ASSERT_EQ_CLEANUP("read", strcmp(buf, VFS_TEST_CONTENTS_3), 0, close(fd2));
+
+	/* Deinit */
 	close(fd2);
-	TC_ASSERT_GT("read", ret, 0);
-	TC_ASSERT_EQ("read", strcmp(buf, VFS_TEST_CONTENTS_3), 0);
+	vfs_unmount();
```

Comment:
There are too many "vfs_unmount()" at the end of TCs, so I'll stop commeting about this here.
Please modify remaining vfs_unmount() if needed.

Reply:
As in the case above, at the point where the function terminates, it is terminated after the deinit operation.